api/health/service.md

```diff
@@ -0,0 +1,214 @@
+---
+tags: [AvalancheGo APIs]
+description: This page is an overview of the Health API associated with AvalancheGo. This API can be used for measuring node health.
+sidebar_label: Health API
+pagination_label: Health API
+---
```

Comment:
This is [markdown front matter](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#markdown-front-matter) that helps enhance the SEO of our site, and controls certain rendered aspects, like the page title, sidebar label, etc.